include/swift/ClangImporter/ClangImporter.h

```diff
@@ -220,11 +220,15 @@ class ClangImporter final : public ClangModuleLoader {
   /// \param path A sequence of (identifier, location) pairs that denote
   /// the dotted module name to load, e.g., AppKit.NSWindow.
   ///
+  /// \param AllowMemoryCache Enables preserving the loaded module in the
+  /// in-memory cache for the next loading attempt.
```

Comment:
The implementation appears to ignore this parameter, so I think we should change this particular `ClangImporter::loadModule` doc comment to reflect reality.

Reply:
Good point, I updated it.